automation_tools/__init__.py

```diff
@@ -918,7 +918,9 @@ def product_install(distribution, create_vm=False, certificate_url=None,
         beta=distribution.endswith('beta'),
         host=host
     )
-    execute(update_packages, warn_only=True)
+
+    # Update the machine
+    update_packages(warn_only=True)
```

Comment:
Yeah my fix was incorrect and the automation failed agan.  I investigarted run and it does not take host parameter.  So your suggested fix will fail as well.
I am testing this instead
```py
with settings(warn_only=True, host_string=host):
        update_packages()
```